src/main/java/org/opensearch/performanceanalyzer/PerformanceAnalyzerApp.java

```diff
@@ -83,7 +83,9 @@
 
     private static final int EXCEPTION_QUEUE_LENGTH = 1;
     public static final String QUERY_URL = "/_plugins/_performanceanalyzer/metrics";
+    public static final String LEGACY_QUERY_URL = "/_opendistro/_performanceanalyzer/metrics";
```

Comment:
Should be `LEGACY_OPENDISTRO_...`

Reply:
I wasn't able to find a way to mark the legacy endpoints as deprecated, but I updated the constant names. We've our integration tests to hit both endpoints.